lib/Command/Test.php

```diff
@@ -44,17 +45,22 @@ class Test extends Command {
 	/** @var TelegramGateway */
 	private $telegramGateway;
 
+        /** @var XMPPGateway */
+        private $xmppGateway;
+
```

Comment:
You will need to run `composer cs:fix` to fix this and other issues that was reported at the CI:

```diff
 > php-cs-fixer fix --dry-run --diff
Loaded config default from "/home/runner/work/twofactor_gateway/twofactor_gateway/.php-cs-fixer.dist.php".
   1) lib/Command/Test.php
      ---------- begin diff ----------
--- /home/runner/work/twofactor_gateway/twofactor_gateway/lib/Command/Test.php
+++ /home/runner/work/twofactor_gateway/twofactor_gateway/lib/Command/Test.php
@@ -45,8 +45,8 @@
 	/** @var TelegramGateway */
 	private $telegramGateway;
 
-        /** @var XMPPGateway */
-        private $xmppGateway;
+	/** @var XMPPGateway */
+	private $xmppGateway;
```

Reply:
Cannot see a difference in the above code (exactly the same lines are removed and then added again).
If I try to do a composer cs:fix as you suggested in my local copy, it stops with an error:
```
> php-cs-fixer fix
sh: 1: php-cs-fixer: not found
```
Commited the change you requested before. Can you run the fix? I am completely new to unit testing.

Reply:
Was only by example, but wasn't a good example, sorry.
running the command `composer cs:fix` will solve all issues about lint.

Reply:
Finally I got it to work ... hope that now everything is fine and can be merged.